src/algebraic_geometry/Spec.lean

```diff
@@ -23,13 +25,9 @@ We define $Spec$ in three consecutive steps, each with more structure than the l
 Additionally, we provide `Spec.to_PresheafedSpace` as a composition of `Spec.to_SheafedSpace` with
 a forgetful functor.
 
-## In progress
+## Related results
 
-Adjunction between `Γ` and `Spec`: Currently, the counit of the adjunction is proven to be a
-natural transformation in `Spec_Γ_naturality`, and realized as a natural isomorphism in
-`Spec_Γ_identity`.
-
-TODO: provide the unit, and prove the triangle identities.
+The adjunction `Γ ⊣ Spec` is constructed in `algebraic_geometry/Gamma_Spec_adjunction.lean`.
```

Comment:
It might be better to keep this change for the final PR that actually adds the adjunction.